topics/proteomics/tutorials/neoantigen-5-variant-annotation/tutorial.md

```diff
@@ -0,0 +1,460 @@
+>
+> 5. Add to each database a tag corresponding to ...
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+{: .hands_on}
+
+
+
+## Mutation to Peptide Mapping with Query Tabular
+
+In this step, we will use the Query Tabular tool to perform a SQL query on a tabular dataset. This tool allows you to filter and extract data from a dataset by applying SQL commands, making it ideal for working with structured data such as CSV or TSV files. You will use it to join tables, select relevant columns, and generate a new output based on the criteria specified in the query. This step helps in refining and organizing the data for further analysis.
+This step extracts information about novel peptides from Frapipe, which primarily includes protein details and the location of the peptide on the protein.
+
+> <hands-on-title> Query Tabular </hands-on-title>
+>
+> 1. {% tool [Query Tabular](toolshed.g2.bx.psu.edu/repos/iuc/query_tabular/query_tabular/3.3.2) %} with the following parameters:
+>    - In *"Database Table"*:
+>        - {% icon param-repeat %} *"Insert Database Table"*
+>            - {% icon param-file %} *"Tabular Dataset for Table"*: `Novel_Peptides_from_PepQuery.tabular` (Input dataset)
+>        - {% icon param-repeat %} *"Insert Database Table"*
+>            - {% icon param-file %} *"Tabular Dataset for Table"*: `Fragpipe-Peptide-Report.tabular` (Input dataset)
+>    - *"SQL Query to generate tabular output"*:
+> ``` sql
+> SELECT t1.c1,t2.c13,t2.c5,t2.c6
+> FROM t1
+> INNER JOIN t2
+> ON t1.c1 = t2.c1
+> ```
+>    - *"include query result column headers"*: `No`
+>
+{: .hands_on}
+
+
+
+## Annotation and Filtering
+
+### Converting delimiters
+In this step, we will use the Convert tool to modify characters in a dataset. Specifically, the tool will be used to convert all instances of pipe characters (|) to another format. The tool helps clean and standardize the data for subsequent processing or analysis. This is often a necessary step when preparing data for tools that require specific formats or when performing tasks such as parsing or file importation.
+
+> <hands-on-title> Convert </hands-on-title>
+>
+> 1. {% tool [Convert](Convert characters1) %} with the following parameters:
+>    - *"Convert all"*: `Pipes`
+>    - {% icon param-file %} *"in Dataset"*: `output` (output of **Query Tabular** {% icon tool %})
+>
+>
+{: .hands_on}
+
+
+> <question-title></question-title>
+>
+> 1. What is the purpose of using the "Convert" tool in this step?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. The "Convert" tool is used to replace or remove specific characters in a dataset, such as pipe characters (|). This ensures that the data conforms to the required format for subsequent analysis or tools. Pipe characters are often used as delimiters in tabular data. Replacing them might be necessary if the data needs to be formatted for a different tool or if the pipe character interferes with parsing in downstream steps.
```

Comment:
the layout was strange in the answer because of the `|` character, that made markdown think you wanted to format it as a table. It is always safer to but characters like that between backticks (I did it for you already, but just so you understand)